pytest_fixtures/core/broker.py

```diff
@@ -22,7 +22,13 @@ def _resolve_deploy_args(args_dict):
 def _default_sat(align_to_satellite):
     """Returns a Satellite object for settings.server.hostname"""
     if settings.server.hostname:
-        return Satellite()
+        hosts = Broker(host_classes={'host': Satellite}).from_inventory(
```

Comment:
@JacobCallahan thoughts here? Almost seems like we need a classmethod instance constructor for satellite.

Reply:
This is currently the correct pattern